website/content/api-docs/acl/auth-methods.mdx

```diff
@@ -0,0 +1,392 @@
+---
+layout: api
+page_title: ACL Auth Methods - HTTP API
+description: The /acl/auth-methods endpoints are used to configure and manage ACL auth methods.
+---
+
+# ACL Auth Methods HTTP API
+
+The `/acl/auth-methods` and `/acl/auth-method` endpoints are used to manage ACL Auth Methods.
+
+## Create Auth Method
+
+This endpoint creates an ACL Auth Method. The request is always forwarded to the
+authoritative region.
+
+| Method | Path               | Produces           |
+| ------ | ------------------ | ------------------ |
+| `POST` | `/acl/auth-method` | `application/json` |
+
+The table below shows this endpoint's support for
+[blocking queries](/api-docs#blocking-queries) and
+[required ACLs](/api-docs#acls).
+
+| Blocking Queries | ACL Required |
+| ---------------- | ------------ |
+| `NO`             | `management` |
+
+### Parameters
+
+- `Name` `(string: <required>)` - Names is the identifier of the ACL Auth
+  Method.  The name can contain alphanumeric characters, dashes, and underscores.
+  This name must be unique and must not exceed 128 characters.
+
+- `Type` `(string: <required>)` - ACL Auth Role SSO identifier. Currently, the
+  only supported Type is "OIDC." 
+
+- `TokenLocality` `(string: <required>)` - Defines whether the ACL Auth Method
+  creates a local or global token when performing SSO login. This field must be
+  set to either "local" or "global"
+
+- `MaxTokenTTL` `(duration: <required>)` - Defines the maximum life of a token created
+  by this method. When set it will initialize the `ExpirationTime` field on all
+  tokens to a value of `Token.CreateTime + AuthMethod.MaxTokenTTL`. This field is
+  not persisted beyond its initial use. Can be specified in the form of `"60s"` or
+  `"5m"` (i.e., 60 seconds or 5 minutes, respectively). 
+
+- `Default` `(bool: false)` - Defines whether this ACL Auth Method is to be
+  set as default when running `nomad login` command. 
+
+- `Config` `(ACLAuthMethodConfig: <required>)` - The raw configuration to use for
+  the auth method.
```

Comment:
I wonder if it is worth expanding this to explain these are specific to the SSO provider, whereas the top level are specific to Nomad?

Reply:
Yeah so I thought about writing something along the lines of: "config differs per auth method" but abandoned it since we only support 1 auth method for now. But yes, mentioning that top-level and auth-specific are different domains is a good idea! 👍